linux/history.md

```diff
@@ -0,0 +1,4 @@
+# Keyman for Linux Version History
+
+## 2018-04-23 
```

Comment:
I think this initial stage is just

```
## 11.0 alpha
```

(skip the date until this goes to 11.x beta).

Also, maybe include what version of KMFL you're importing.